src/accelerate/accelerator.py

```diff
@@ -2580,6 +2580,10 @@ def save_model(
 
         os.makedirs(save_directory, exist_ok=True)
 
+        for param in model.parameters():
+            if param.device == torch.device("meta"):
+                raise RuntimeError("You can't save the model since some parameters are on the meta device.")
```

Comment:
```suggestion
        if any(param.device == torch.device("meta") for param in model.parameters()):
            raise RuntimeError("You can't save the model since some parameters are on the meta device.")
```

Reply:
better indeed =)